doc/man1/s_server.pod

```diff
@@ -125,14 +130,34 @@ manual page.
 
 Print out a usage message.
 
-=item B<-accept port>
+=item B<-port port>
 
 The TCP port to listen on for connections. If not specified 4433 is used.
 
+=item B<-accept val>
+
+The TCP optional host and port to listen on for connections. If not specified 4433 is used.
```

Comment:
"The optional TCP host..."  And "specified, "*:4433" is used"  (Not quotes around the *:4433.